libraries/lib-track/Track.cpp

```diff
@@ -51,7 +51,7 @@ Track::Track(const Track& orig, ProtectedCreationArg&&)
 void Track::Init(const Track &orig)
 {
    mId = orig.mId;
-
+   mProjectTempo = orig.mProjectTempo;
```

Comment:
I think this omission should have been detected sooner, when testing #4867 or #4870.

I think it will affect copy/paste and undo/redo for all track types.

I will add to the QA checklist.